addon/webextension/background/startBackground.js

```diff
@@ -69,6 +71,30 @@ this.startBackground = (function() {
     return true;
   });
 
+  // We delay this check (by CHECK_MIGRATION_DELAY) just to avoid piling too
+  // many things onto browser/add-on startup
+  setTimeout(() => {
```

Comment:
Note that `requestIdleCallback` provides a `timeout` parameter that we can use to guarantee execution by a particular point in time